mewarpx/tests/test_capacitive_discharge.py

```diff
@@ -0,0 +1,366 @@
+"""Test full 1D diode run with diagnostics."""
+# [[[TODO]]] NOT TOUCHED, JUST COPIED FROM WARP, SO FAR
+from __future__ import division
+
+from builtins import range
+# import os
+# import sys
+
+import pytest
+import numpy as np
+
+from pywarpx import picmi
+# import pywarpx
+# import mewarpx
+
+# Necessary to load Warp as library (because Warp processes arguments even when
+# loaded as a module!!!)
+# del sys.argv[1:]
+
+# import warp
+# from warp.data_dumping.openpmd_diag import ParticleDiagnostic
+
+# from metools import analysis, diags
+# from metools import init_restart_util, runtools, util, warputil
+from mewarpx.setups_store import diode_setup
+
+
+constants = picmi.constants
+
+##########################
+# physics parameters
+##########################
+
+D_CA = 0.067 # m
+
+N_INERT = 9.64e20 # m^-3
+T_INERT = 300.0 # K
+
+FREQ = 13.56e6 # MHz
+
+VOLTAGE = 450.0
+
+M_ION = 6.67e-27 # kg
+
+PLASMA_DENSITY = 2.56e14 # m^-3
+T_ELEC = 30000.0 # K
+
+##########################
+# numerics parameters
+##########################
+
+# --- Grid
+nx = 128
+nz = 16
+
+xmin = 0.0
+zmin = 0.0
+xmax = D_CA
+zmax = D_CA / nx * nz
+
+number_per_cell_each_dim = [32, 16]
+
+DT = 1.0 / (400 * FREQ)
+
+# Total simulation time in seconds
+TOTAL_TIME = 10.0 * DT # 1280 / FREQ
+# Time (in seconds) between diagnostic evaluations
+DIAG_INTERVAL = 2.0 * DT # 32 / FREQ
+
+# --- Number of time steps
+max_steps = int(TOTAL_TIME / DT)
+diag_steps = int(DIAG_INTERVAL / DT)
+diagnostic_intervals = "::%i" % diag_steps #"%i:" % (max_steps - diag_steps + 1)
+
+print('Setting up simulation with')
+print('  dt = %.3e s' % DT)
+print('  Total time = %.3e s (%i timesteps)' % (TOTAL_TIME, max_steps))
+print('  Diag time = %.3e s (%i timesteps)' % (DIAG_INTERVAL, diag_steps))
+
+# mwxutil.init_libwarpx(ndim=2, rz=False)
+@pytest.mark.parametrize(
+    ("name"),
+    [
+        'Run1D_alldiags_2DScraper',
```

Comment:
Change these names - alldiags is no longer relevant, but make these descriptive 2D vs RZ vs 3D